tool.rkt

```diff
@@ -2,7 +2,7 @@
 (require
   (for-syntax racket/base) ; for help menu
   drracket/tool ; necessary to build a drracket plugin
-  framework ; for preferences (too heavy a package?)
+  framework
```

Comment:
The existing use was not, in fact, for packages, though this PR does start using `framework/preferences` elsewhere.

Reply:
That's a relic: 
I used to use the preference system in the first version of Quickscript (called script-plugin) and can still be seen in the first commit:
https://github.com/Metaxal/quickscript/blob/ec6b3e0a0eae0578fcc1a60ffb62aaf92909f6ab/tool.rkt
The library replaced the preferences on the next commit.